apps/dashboard/src/constants/schemas.ts

```diff
@@ -49,3 +49,31 @@ const AddressSchema = z.custom<string>(
 export const AddressOrEnsSchema = z.union([AddressSchema, EnsSchema], {
   invalid_type_error: "Provided value was not a valid address or ENS name",
 });
+
+const FileOrBufferUnionSchema =
+  typeof window !== "undefined"
+    ? (z.instanceof(File) as z.ZodType<InstanceType<typeof File>>)
+    : // @fixme, this is a hack to make browser happy for now
+      (z.instanceof(Buffer) as z.ZodTypeAny);
+
+const FileOrBufferSchema = z.union([
+  FileOrBufferUnionSchema,
+  z.object({
+    data: z.union([FileOrBufferUnionSchema, z.string()]),
+    name: z.string(),
+  }),
+]);
```

Comment:
I do not think you need this in dashboard, all you need is File or string, there are no `Buffer` in the frontend here? Can we simplify it based on that?